### PR TITLE
feat: Add spec and parser for sos_commands/logs/journalctl_--no-pager…

### PR DESCRIPTION
**Pull request by qinpingli**
… file

Signed-off-by: Qin Ping <piqin@redhat.com>

### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

There are some user cases, the sosreport is collected after the host reboot, then the kernel panic info is not included in the journal since boot. Detail info please see: INSIGHTOCP-532 card.

### CODE REVIEW

**Comment by qinpingli** [comment]
@xiangce Please help review, thanks!

**Comment by qinpingli** [comment]
Will move it to the ccx core.

**Comment by qinpingli** [comment]
Reopen it, for this spec and parser is a shared parse between GSS rule and CCX rule.

File: insights/parsers/journal_all.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Sample log lines::
+
+        -- Logs begin at Wed 2017-02-08 15:18:00 CET, end at Tue 2017-09-19 09:25:27 CEST. --
+        May 18 15:13:34 lxc-rhel68-sat56 jabberd/sm[11057]: session started: jid=rhn-dispatcher-sat@lxc-rhel6-sat56.redhat.com/superclient
+        May 18 15:13:36 lxc-rhel68-sat56 wrapper[11375]: --> Wrapper Started as Daemon
+        May 18 15:13:36 lxc-rhel68-sat56 wrapper[11375]: Launching a JVM...
+        May 18 15:24:28 lxc-rhel68-sat56 yum[11597]: Installed: lynx-2.8.6-27.el6.x86_64
+        May 18 15:36:19 lxc-rhel68-sat56 yum[11954]: Updated: sos-3.2-40.el6.noarch
+
+    Note:
```
Comment (xiangce) [inline insights/parsers/journal_all.py:25]: This format of "Note" is a good approach (in the original `journal_since_boot`), would you please modify this one (and only this one, doesn't mean the one in `journal_since_boot`) according to the `Sphinx` style?

See:  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/journal_all.py (hunk 1 above)
Comment (qinpingli) [reply insights/parsers/journal_all.py]: Thanks! Updated.

File: insights/parsers/journal_all.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+"""
+JournalAll file ``/sos_commands/logs/journalctl_--no-pager``
+=========================================================================
```
[the remark this answers is not in the document]
Comment (xiangce) [reply insights/parsers/journal_all.py:3]: @qinpingli - you might miss this one. And you also need to add this parser to the doc build entry:

- docs/shared_parsers_catalog/journal_all.rst

[the remark this answers is not in the document]
Comment (qinpingli) [reply insights/parsers/journal_all.py]: @xiangce Thanks! Updated.

**Review by xiangce — APPROVED** [review]

**Pull request merged**